boards/arm/pan1783_evb/pan1783_evb_cpunet.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: pan1783_evb_cpunet
+name: PAN1783-EVB-network-MCU
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+ram: 64
+flash: 256
+supported:
+  - watchdog
+  - gpio
```

Comment:
Add `vendor: panasonic`.

Reply:
Sorry, I could have thought of that while changing first yaml file.